src/main/java/edu/ucsb/nceas/mdqengine/scheduler/RequestReportJob.java

```diff
@@ -182,233 +197,265 @@ public void execute(JobExecutionContext context)
 
         // Don't know node type yet from the id, so have to manually check if it's a CN
         Boolean isCN = DataONE.isCN(nodeServiceUrl);
-        if(isCN) {
+        if (isCN) {
             cnNode = new MultipartCNode(mrc, nodeServiceUrl, session);
         } else {
             mnNode = new MultipartMNode(mrc, nodeServiceUrl, session);
         }
 
         // Get a connection to the database
-        MDQStore store = null;
 
-        try {
-            store = new DatabaseStore();
-        } catch (Exception e) {
-            e.printStackTrace();
-            throw new JobExecutionException("Cannot create store, unable to schedule job", e);
-        }
+        try (DatabaseStore store = new DatabaseStore();) {
```

Comment:
semicolon unnecessary here

Reply:
call to printStackTrace() on line 214 should probably be replaced with a logger call

Reply:
Also, your `try-catch` on lines 211/213 could probably be removed, since this whole try-with-resources block as a `catch (Exception e)` that logs and throws exactly the same thing (see line 422)